tools/testing/test_selections.py

```diff
@@ -30,43 +32,75 @@
         NUM_PROCS = 1
 
 
+class ShardedTest(NamedTuple):
+    name: str
+    shard: int
+    num_shards: int
+    time: Optional[float]
+
+    def __str__(self) -> str:
+        if self.num_shards == 1:
+            return self.name
+        return f"{self.name} {self.shard}/{self.num_shards}"
+
+
 class ShardJob:
     def __init__(self, test_times: Dict[str, float]):
```

Comment:
Hmm, something is a bit weird here `test_times` is a param but it's not used now after your change.  I wonder why some linters don't bark about unused variable here